content/fr/docs/reference/kubectl/quick-reference.md

```diff
@@ -0,0 +1,504 @@
+---
+title: Aide-mémoire de kubectl
+content_type: concept
+weight: 10 # highlight it
+card:
+  name: tasks
+  weight: 10
+---
+
+<!-- overview -->
+
+Cette page contient une liste des commandes et options fréquemment utilisées de `kubectl`.
+{{< note >}}
+Ces instructions concernent Kubernetes v{{< skew currentVersion >}}. Pour vérifier la version, utilisez la commande `kubectl version`.
+{{< /note >}}
+<!-- body -->
+
+## Autocomplétion de Kubectl
+
+### BASH
+
+```bash
+source <(kubectl completion bash) # set up autocomplete in bash into the current shell, bash-completion package should be installed first.
+echo "source <(kubectl completion bash)" >> ~/.bashrc # add autocomplete permanently to your bash shell.
+```
+
+Vous pouvez également utiliser un alias pour kubectl qui fonctionne aussi avec l'autocomplétion.
+
+```bash
+alias k=kubectl
+complete -o default -F __start_kubectl k
+```
+
+### ZSH
+
+```bash
+source <(kubectl completion zsh)  # set up autocomplete in zsh into the current shell
+echo '[[ $commands[kubectl] ]] && source <(kubectl completion zsh)' >> ~/.zshrc # add autocomplete permanently to your zsh shell
+```
+
+### FISH
+
+{{< note >}}
+Requires kubectl version 1.23 or above.
+{{< /note >}}
+
+```bash
+echo 'kubectl completion fish | source' > ~/.config/fish/completions/kubectl.fish && source ~/.config/fish/completions/kubectl.fish
+```
+
+### Remarque concernant `--all-namespaces`
+
+L'utilisation de `--all-namespaces` est assez fréquente, alors sachez qu'il existe un raccourci pour cela :
```

Comment:
```suggestion
L'utilisation de `--all-namespaces` (tous les espaces de nommage) est assez fréquente, alors sachez qu'il existe un raccourci pour cela :
```
?

Reply:
--all-namespaces is a command. I don't think it's convenient to translate it, except maybe for absolute beginners, so I will commit it.